### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ssiccha**
I'm using NVIM v0.8.0-dev+177-gcf68f0a51 and got the same errors as in https://github.com/nvim-neorg/neorg/issues/559. It's nice that that issue is pinned, so that I found the issue relatively quickly. Stating this in the README is still a bit easier to discover.

### CODE REVIEW

**Comment by max397574** [comment]
you should use 0.8 stable then it should work. You're on an older 0.8 version which perhaps doesn't have the patch yet.

**Comment by ssiccha** [comment]
>you should use 0.8 stable then it should work. You're on an older 0.8 version which perhaps doesn't have the patch yet.

Good to know, thanks! Then I'll update my Neovim and can get rid of the tag fix. In that case I'd reword the PR slightly.

**Comment by max397574** [comment]
imo the only thing that could perhaps be changed is "neorg requires at least nvim 0.8 **stable** to operate"

**Comment by ssiccha** [comment]
Ah, I see. I wasn't aware that recently there was an official 0.8 release.

**Comment by ssiccha** [comment]
In that case I'll close the PR. I forgot that I installed a dev version of Neovim. :grin:
